### PR TITLE
enhance: RBAC Custom Privilege Group API

### PR DESCRIPTION
**Pull request by shaoting-huang**
issue: https://github.com/milvus-io/milvus/issues/37031

### CODE REVIEW

File: pymilvus/milvus_client/milvus_client.py [hunk 1]
```diff
@@ -1152,3 +1152,108 @@ def get_server_version(
...
+    ) -> List[str]:
+        """List all privilege groups.
+
+        Args:
+            timeout (``float``, optional): An optional duration of time in seconds to allow
+                for the RPC. When timeout is set to None, client waits until server response
+                or error occur.
+
+        Returns:
+            List[str]: A list of all privilege groups and its privileges.
```
Comment (weiliu1031) [inline pymilvus/milvus_client/milvus_client.py:1209]: how to distinguish privilege groups and its privileges from a sting list?

**Review by XuanYang-cn — APPROVED** [review]

**Comment by XuanYang-cn** [comment]
/unapprove
@shaoting-huang please fix the code checker

**Review by XuanYang-cn — APPROVED** [review]
/lgtm

**Pull request merged**